lib/utils/resolveObject.js

```diff
@@ -142,16 +158,10 @@ function selfRef(string, obj) {
   // If the reference doesn't change then it means
   // we didn't find it in the object
   if (ref === obj) {
-    throw new Error('Reference doesn\'t exist: ' + context + ' tries to reference ' + string + ', which is not defined');
+    handleErrors.add(ERROR_TYPE, 'Reference doesn\'t exist: ' + context + ' tries to reference ' + string + ', which is not defined');
```

Comment:
do you have any idea if in this kind of errors it's possible to know also in which **file** the missing reference (or the collision) is happening? often doing refactoring of the properties it happened to me to see the error in the console, but I had to figure out which file was causing it. looking at the code, is not so easy/straightforward but maybe there is a simple way and I can't see it

Reply:
At this point, no, because we have already merged all the token JSON files together.. I mean we could find it, but we would have to go through the files again.

Reply:
I thought about this as well. I went through the code and I could actually show where I found the conflict. However, without significant additional work I would NOT know where the first occurrence was. I thought there were three options:

1. Do the significant additional work to report the positions of the conflicts, and make this PR even bigger/more confusing...
2. Only report the position of the collision. User will have to do a file search for the original source of the colliding properties.
3. Report nothing. User will have to do a file search for both the original and colliding properties.

Since the difference in experience between #2 and #3 seemed minimal, I decided to not go with #2. As I alluded to in my first paragraph, #1 seemed like too much to add to this PR.

I would still like to tackle this, but maybe after we get it merged? Should we add an issue to track it?

Reply:
@chazzmoney I didn't mean to do it in this PR, I was just thinking out loud, to see if it's something that we may consider in the future. So whatever you choose to do, is ok for me.

Reply:
Added as issue #166